sdk/identity/azure-identity/src/main/java/com/azure/identity/SharedTokenCacheCredential.java

```diff
@@ -80,7 +82,7 @@
                     .authority(authorityUrl)
                     .setTokenCacheAccessAspect(accessAspect)
                     .build();
-            } catch (MalformedURLException e) {
+            } catch (IOException | PlatformNotSupportedException e) {
```

Comment:
Can you catch `Exception` here? We don't want any exception to be directly thrown

Reply:
done